crud/common/schema.lua

```diff
@@ -135,7 +135,40 @@ local function get_space_schema_hash(space)
     return digest.murmur(msgpack.encode(space_info))
 end
 
-local function filter_result_fields(tuple, field_names)
+function schema.truncate_tuple_trailing_fields(tuple, field_names)
+    dev_checks('table|cdata', '?table')
```

Comment:
`cdata` -> `tuple`

Reply:
Also it won't work for box.tuples - box.tuple is immutable.

```
tarantool> box.tuple.new({1, 2, 3})[3] = nil
---
- error: '[string "box.tuple.new({1, 2, 3})[3] = nil"]:1: ''struct tuple'' cannot
    be indexed with ''number'''
...
```

It works only for tables.

Reply:
If you want to support it also for tuples consider `truncate` function. But I don't think that currently it's needed.